client/src/components/pages/CreateQuiz.js

```diff
@@ -438,36 +440,41 @@ function CreateQuiz(props) {
         }
         
         if(completed) {
-            setTimeout(() => navigate('/'), 4000);
+            setTimeout(() => navigate('/quizsend'), 4000);
```

Comment:
I think this is unnecessary and would be better to navigate to next page right away.

Reply:
I think there is a timeout because of the notification I set when the user successfully creates the quiz. I'll review this more if I want to remove that notification